src/herder/TxSetFrame.cpp

```diff
@@ -681,27 +669,56 @@ TxSetFrame::checkValid(Application& app, uint64_t lowerBoundCloseTimeOffset,
 }
 
 size_t
-TxSetFrame::size(LedgerHeader const& lh) const
+TxSetFrame::size(LedgerHeader const& lh, std::optional<Phase> phase) const
 {
-    return protocolVersionStartsFrom(lh.ledgerVersion, ProtocolVersion::V_11)
-               ? sizeOp()
-               : sizeTx();
+    size_t sz = 0;
+#ifdef ENABLE_NEXT_PROTOCOL_VERSION_UNSAFE_FOR_PRODUCTION
+    if (!phase && phase == Phase::SOROBAN)
+    {
+        sz += sizeTx(Phase::SOROBAN);
```

Comment:
I think it's better to use `sizeOp` here for the sake of consistency (I realize they're equivalent, that's just for clarity)

Reply:
Done.